cmd/secrets/backend_selector.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build secrets
+
+package secrets
+
+import (
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/apiserver"
+	"time"
```

Comment:
Fixed :+1:

I agree with the structure, but I think it should be enforced by the CI, otherwise it's going to be difficult to keep it consistent across al the codebase.